modules/s3/s3codecs/pdf.py

```diff
@@ -90,6 +91,17 @@
 PDF_WIDTH = 0
 PDF_HEIGHT = 1
 
+if current.response.s3.language == "tr":
```

Comment:
If just that, then it could be a tuple, not a storage.
A dict/Storage allows more complex language -> font mappings.
If your anlysis shows that actually we have just 2 cases: Unicodie & normal, then the tuple is fine.
If there other languages which require other fonts (I thought we already had a different font being required for Japanese) then we need the more structured type.